package/scripts/params.py

```diff
@@ -71,11 +72,30 @@
     hdfs_namespace = '/' + root_namespace
 else:
     hdfs_namespace = map_cdap_site['hdfs.namespace']
-hdfs_user = map_cdap_site['hdfs.user']
-# TODO: Fix this hack -- check if we're still cdap_user
-hdfs_user = cdap_user
 kafka_log_dir = map_cdap_site['kafka.log.dir']
 
+# Kerberos stuff
+hdfs_user_keytab = config['configurations']['hadoop-env']['hdfs_user_keytab']
+hdfs_user = config['configurations']['hadoop-env']['hdfs_user']
+hdfs_principal_name = config['configurations']['hadoop-env']['hdfs_principal_name']
+kinit_path_local = get_kinit_path(default('/configurations/kerberos-env/executable_search_paths', None))
+cdap_principal_name = config['configurations']['cdap-env']['cdap_principal_name']
+cdap_user_keytab = config['configurations']['cdap-env']['cdap_user_keytab']
+
+if security_enabled:
+    _hostname_lower = config['hostname'].lower()
```

Comment:
maybe can just use ``hostname.lower()`` since ``hostname`` is already defined [above](https://github.com/caskdata/cdap-ambari-service/blob/fd90b7678c39b608a4a16d7e282b8e4e2836a929/package/scripts/params.py#L31)